src/Entity/Event.php

```diff
@@ -31,6 +32,9 @@ class Event extends BaseEvent implements UserDocumentInterface
      */
     private $committee;
 
+    /**
+     * @JMS\Exclude
```

Comment:
Because JMS tries to serialize this from the parent, and this does not exist in the parent, and it's a weird behavior and this is the only workaround i found.

This field is doctrine specific so JMS shouldn't know about it.

Reply:
Alright !